types/quill/index.d.ts

```diff
@@ -84,9 +84,9 @@ export interface DeltaStatic {
     concat(other: DeltaStatic): DeltaStatic;
     diff(other: DeltaStatic, index?: number): DeltaStatic;
     eachLine(predicate: (line: DeltaStatic, attributes: StringMap, idx: number) => any, newline?: string): DeltaStatic;
-    transform(index: number): number;
+    transform(index: number, priority?: boolean): number;
```

Comment:
According to [the doc](https://github.com/quilljs/delta/#methods-15), priority is only on `transform(other: DeltaStatic, priority: boolean)`.